webpack/assets/javascripts/react_app/components/PasswordStrength/PasswordStrength.scss

```diff
@@ -11,13 +11,18 @@
   }
 
   .ReactPasswordStrength-strength-desc {
-    padding: 6px 10px;
+    padding: 2px 10px;
     right: -62.5%;
     font-size: 12px;
     width: 60%;
     text-align: left;
     font-style: normal;
     font-family: $font-family-base;
+
+    @media (max-width: 992px) {
```

Comment:
You can use the variable: `$screen-sm-max` from bootstrap:
https://github.com/twbs/bootstrap/blob/master/less/variables.less#L316